src/main/java/dukeApp/files/DateTime.java

```diff
@@ -0,0 +1,32 @@
+package dukeApp.files;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class DateTime {
+    protected String date;
+    protected String time;
+
+    public DateTime() {
+    }
+
+    /**
+     * Convert the date string into a date format
+     * @param date date in String
+     * @return date of a specific format
+     */
+    public String convertDate(String date) {
+        LocalDate d = LocalDate.parse(date);
+        return d.format(DateTimeFormatter.ofPattern("MMM d yyyy"));
```

Comment:
Perhaps you can handle the error that occurs when the DateTimeFormatter cannot convert into a valid date format due to invalid date string